Backend/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "ConnectionStrings": {
```

Comment:
Think this may have been needed for EF.

I'm aware it is bad practice to push such information to repos but storing our information in our repo is ultimately the easiest way.